src/Microsoft.AspNet.Identity/LockoutOptions.cs

```diff
@@ -10,7 +10,7 @@ public class LockoutOptions
         /// <summary>
         ///     If true, will enable user lockout when users are created
         /// </summary>
-        public bool EnabledByDefault { get; set; } = false;
+        public bool AllowedForNewUsers { get; set; } = false;
```

Comment:
In a local change that I forgot to push :) Was lost in the merge from the dev branch...